tests/bluetooth/audio/ascs/mocks/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+#
```

Comment:
IMO it would be nice to separate .c and .h files in their own directory src and include
Keep in mind also that sooner or later we should relocate the unittest for host, ascs and controller.

Reply:
done